.github/workflows/docs.yml

```diff
@@ -1,8 +1,8 @@
-name: Deploy Evolve docs
+name: Deploy Docs 
 
 on: 
   repository_dispatch:
-    types: [deploy-evolve-docs]
+    types: [deploy-docs, deploy-evolve-docs]
```

Comment:
I've added this so that I could swap them over via GITHUB secrets. Will clean the other one when done.